frontend/src/components/AccountReceiveTable.vue

```diff
@@ -472,40 +473,28 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
   };
 
   // Table formatters and helpers
-  const formatDate = (timestamp: number) => date.formatDate(timestamp, 'YYYY-MM-DD');
-  const formatTime = (timestamp: number) => date.formatDate(timestamp, 'h:mm A');
   const isNativeToken = (tokenAddress: string) => tokenAddress === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
-  const getTokenInfo = (tokenAddress: string) => tokens.value.filter((token) => token.address === tokenAddress)[0];
+  const getTokenInfo = (tokenAddress: string) => tokens.value.filter(token => token.address === tokenAddress)[0];
```

Comment:
Can you regenerate the lockfile (mine differs after running `yarn`) and run `yarn prettier` (some changes in here seem to have been made by your editor on save)